src/views/SpecificQuery.js

```diff
@@ -156,8 +176,20 @@ export const SpecificQuery = () => {
                     <DialogContentText>{status} specific query {status === 'Adding' ? newAlias : aliasToDelete}?</DialogContentText>
```

Comment:
please add single quotes or square brackets around the displayed alias.

Reply:
Added single quotes wherever we're displaying an alias on a pop-up box